plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js

```diff
@@ -0,0 +1,82 @@
+const { test, expect } = require( '@playwright/test' );
+const { exec } = require( 'child_process' );
+
+test.describe( 'WC Home Task List >', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.beforeAll( async ( {} ) => {
+		return new Promise( ( resolve, reject ) => {
+			const deleteCommand = `wp-env run tests-cli wp option delete woocommerce_task_list_hidden_lists --allow-root`;
+			exec( deleteCommand, ( error, stdout, stderr ) => {
+				if ( error ) {
+					console.error(
+						`Error deleting option: ${ error.message }`
+					);
+					return reject( error );
+				}
+				if ( stderr && ! stderr.includes( 'Ran `wp option delete' ) ) {
+					console.error( `Error output: ${ stderr }` );
+					return reject( new Error( stderr ) );
+				}
+				console.log( 'Option deleted successfully.' );
+				resolve( stdout );
+			} );
+		} );
+	} );
+
+	test.afterAll( async ( {} ) => {
+		return new Promise( ( resolve, reject ) => {
+			const deleteCommand = `wp-env run tests-cli wp option delete woocommerce_task_list_hidden_lists --allow-root`;
+			exec( deleteCommand, ( error, stdout, stderr ) => {
+				if ( error ) {
+					console.error(
+						`Error deleting option: ${ error.message }`
+					);
+					return reject( error );
+				}
+				if ( stderr && ! stderr.includes( 'Ran `wp option delete' ) ) {
+					console.error( `Error output: ${ stderr }` );
+					return reject( new Error( stderr ) );
+				}
+				console.log( 'Option deleted successfully.' );
+				resolve( stdout );
+			} );
+		} );
+	} );
+
+	test(
+		' Can hide the task list',
+		{ tag: [ '@skip-on-default-pressable', '@skip-on-default-wpcom' ] },
+		async ( { page } ) => {
+			await test.step( 'Load the WC Admin page', async () => {
+				await page.goto( '/wp-admin/admin.php?page=wc-admin' );
```

Comment:
Following changes from [this PR](https://github.com/woocommerce/woocommerce/pull/53228) and supporting multisite setup, URLs should start without `/` or with `./`

Reply:
URL updated.  Thanks for that @Stojdza